news/2017-12-03-osu-idol-2017-finals-community-voting-results.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "osu!idol 2017 Finals - Community Voting Results!"
+date: 2017-12-03 12:00:00 +0000
+---
+
+We have reached another year of osu!idol to an end. With a spectacular 12281 votes in total, this beats last year's voting by a lightyear! Thanks everyone for taking part in the [community voting](https://osu.ppy.sh/community/contests/57)! Now let's get to what we're all here for - results!
```

Comment:
That sentence is poorly worded. You can't "reach something to an end". Just make it "Another year of osu!idol comes to an end".

Reply:
Sure